verilog/analysis/symbol_table_test.cc

```diff
@@ -7597,7 +7597,12 @@ TEST(BuildSymbolTableTest, ModuleInstancesFromProjectMissingFile) {
   symbol_table.BuildSingleTranslationUnit("file/not/found.txt",
                                           &build_diagnostics);
   ASSERT_FALSE(build_diagnostics.empty());
+#ifdef _WIN32
```

Comment:
If we change this code to compare the status code instead of the message it should hopefully be platform independent:

```
EXPECT_EQ(build_diagnostics.front().code(), absl::StatusCode::kNotFound);
```